playbooks/gluster-cluster/gluster_vol_4x2.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Do we need a 2x2 and 4x2 yml files? I didn't see any difference except for the number of bricks created. I think this can be address by adequate comments for each section in the cluster-vars file?

Reply:
I added 4x2 to show how the list can be extended. I think we should have this, unless it causes any confusion to the user.
What do you think?